task-manager/src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
```

Comment:
What does the `expect` function do?

Reply:
It is used to create assertions in our tests. The expect function is used to assert that a value meets certain conditions. It takes a single argument, which is the value we want to test, and returns an "expectation" object. This object contains various matcher methods that we can use to assert different conditions.